docs/blog/posts/volumes-on-runpod.md

```diff
@@ -0,0 +1,139 @@
+
+```yaml
+type: service
+name: llama31-service-tgi
+
+replicas: 1..2
+scaling:
+  metric: rps
+  target: 30
+  
+image: ghcr.io/huggingface/text-generation-inference:latest
+env:
+  - HUGGING_FACE_HUB_TOKEN
+  - MODEL_ID=meta-llama/Meta-Llama-3.1-8B-Instruct
+  - MAX_INPUT_LENGTH=4000
+  - MAX_TOTAL_TOKENS=4096
+commands:
+  - text-generation-launcher
+port: 80
+
+spot_policy: auto
+
+resources:
+  gpu: 24GB
+
+model:
+  format: openai
+  type: chat
+  name: meta-llama/Meta-Llama-3.1-8B-Instruct
+```
+
+</div>
+
+When you run `dstack apply`, it creates a public endpoint with one service replica. dstack will then automatically scale
+the service by adjusting the number of replicas based on traffic.
+
+When starting each replica, `text-generation-launcher` downloads the model to the `/data` folder. For Llama 3.1 8B, this
+usually takes under a minute, but larger models may take longer. Repeated downloads can significantly affect
+auto-scaling efficiency.
+
+Great news: RunPod supports network volumes, which we can use for caching models across multiple replicas.
+
+With dstack, you can create a RunPod volume using the following configuration:
+
+<div editor-title="examples/mist/volumes/runpod.dstack.yml">
+
+```yaml
+type: volume
+name: llama31-volume
+
+backend: runpod
+region: EU-SE-1
+
+# Required size
+size: 100GB
+```
+
+</div>
+
+Go ahead and create it via `dstack apply`:
+
+<div class="termy">
+
+```shell
+$ dstack apply -f examples/mist/volumes/runpod.dstack.yml
+```
+
+</div>
+
+Once the volume is created, attach it to your service by updating the configuration file and mapping the 
+volume name to the `/data` path.
+
+<div editor-title="examples/llms/llama31/tgi/service.dstack.yml">
+
+```yaml
+type: service
+name: llama31-service-tgi
+
+replicas: 1..2
+scaling:
+  metric: rps
+  target: 30
+  
+volumes:
+ - name: llama31-volume
+   path: /data
+  
+image: ghcr.io/huggingface/text-generation-inference:latest
+env:
+  - HUGGING_FACE_HUB_TOKEN
+  - MODEL_ID=meta-llama/Meta-Llama-3.1-8B-Instruct
+  - MAX_INPUT_LENGTH=4000
+  - MAX_TOTAL_TOKENS=4096
+commands:
+  - text-generation-launcher
+port: 80
+
+spot_policy: auto
+
+resources:
+  gpu: 24GB
+  
+model:
+  format: openai
+  type: chat
+  name: meta-llama/Meta-Llama-3.1-8B-Instruct
+```
+
+</div>
+
+In this case, `dstack` attaches the specified volume to each new replica. This ensures the model is downloaded only
+once, reducing cold start time in proportion to the model size.
+
+Using [volumes](../../docs/concepts/volumes.md) can optimize not only inference cold start times but also the loading of data and model checkpoints during
+training and fine-tuning, e.g., via [tasks](../../docs/tasks.md) or [dev environments](../../docs/dev-environments.md).
+
+RunPod, in turn, optimizes cold starts through aggressive caching of Docker images. Starting a new instance with RunPod,
+especially if you have used a specific Docker image before, is exceptionally fast.
+
```

Comment:
I'd avoid words like aggressive and exceptionally fast since it screams marketing and is not really necessary to "sell" the feature. Better mention that a really cool feature about RunPod is that volumes can be attached to multiple instance simultaneously. This allows using just one volume for any number of replicas.